src/AureliaPlugin.ts

```diff
@@ -166,7 +166,7 @@ export class AureliaPlugin {
     if (!opts.noHtmlLoader) {
       // Ensure that we trace HTML dependencies (always required because of 3rd party libs)
       let module = compiler.options.module;
-      let rules = module.rules || (module.rules = []);
+      let rules = module.rules || module.loaders || (module.rules = []);
```

Comment:
I would say this should rather be Webpack's choice to strongly deprecate and notify users rather than ours.
I'm just trying to make the plugin play along nicely with whatever Webpack supports. 
At this point -- although I can find it in the new docs -- the old `loaders` property is still supported by Webpack 2.3.